set.mm

```diff
@@ -804812,6 +804834,214 @@ edge function instead of the edges themselves (see also ~ clnbgrval ).
   $}
 
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Semiclosed and semiopen neighborhoods (experimental)
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+
+  We have already definitions for open and closed neighborhoods of a vertex,
+  which differs only in the fact that the first never contains the vertex,
+  and the latter always contains the vertex. One of these definitions, however,
+  cannot be simply derived from the other.  This would be possible if a
+  definition of a semiclosed neighborhood was available, see ~ dfsclnbgr2 .
+  The definitions for open and closed neighborhoods could be derived from such
+  a more simple, but otherwise probably useless definition, see ~ dfnbgr5 and
+  ~ dfclnbgr5 . Depending on the existence of certain edges, a vertex belongs
+  to its semiclosed neighborhood or not.
+
+  An alternate approach coukd have been to introduce semiopen neighborhoods,
```

Comment:
```suggestion
  An alternate approach could have been to introduce semiopen neighborhoods,
```
typo

```suggestion
  An alternate approach is to introduce semiopen neighborhoods,
```
alternative, since the alternate approach is shown/proven